pkg/cluster/kubernetes_cluster.go

```diff
@@ -259,6 +260,11 @@ func (c *KubernetesCluster) createTokenSecretIfNeeded(ctx context.Context, names
 
 	_, err = c.clientset.CoreV1().Secrets(namespace).Create(ctx, secret, metav1.CreateOptions{})
 	tokenSecretNameTmplEntry := localize.NewEntry("Name", tokenSecretName)
+
+	if err.Error() == "Unauthorized" {
```

Comment:
This is not a reliable way to check unauthorized error. As recommended before let's learn a little bit how kubernetes SDK works.

Example of error handling:
https://github.com/slintes/kubernetes/blob/master/staging/src/k8s.io/client-go/examples/out-of-cluster-client-configuration/main.go#L67-L77

Reply:
Also. We are handling error only in one place. Error handling should be done for each call.

404 - Operator is not installed
403 - No access to cluster (token outdated)
401 - Login required
etc.